project-base/storefront/graphql/requests/orders/queries/OrdersCountQuery.graphql

```diff
@@ -0,0 +1,5 @@
+query OrdersCountQuery($after: String, $first: Int) {
+    orders(after: $after, first: $first) {
+        totalCount
+    }
+}
```

Comment:
This is not very optimal. Did you talk this through with some BE dev? I think they would suggest a different approach. Please, talk this through with somebody.

Reply:
Ohh, I forgot to delete it, I don't need it anymore, thanks